app/views/layouts/application.haml

```diff
@@ -22,7 +22,7 @@
       %meta{property: 'og:description', content: @description}
 
     = csrf_meta_tags
-    %script{type: "text/javascript"}
```

Comment:
mostly unrelated change, except that i realized while working on this change that `type: "text/javascript"}` is not needed

Reply:
nevermind; this awesome answer explained how to do it: https://stackoverflow.com/a/63984111/4009384

Reply:
actually, that answer doesn't quite work, i think, because the font can load before the script gets parsed, in which case the `load` event handler doesn't get called, and so the font doesn't get registered / converted to a stylesheet. i fixed it by manually adding the `href` attribute after registering the load listener. a lot of work is required to squeeze out that last 1% of security and performance together...